src/renderer/config.js

```diff
@@ -25,22 +25,18 @@ function getStaticPath() {
   // Instead, it includes an unexpected `node_modules` path, specifically:
   // node_modules/electron/dist/Electron.app/Contents/Resources/static
   if (isDevelopment) {
-    return path.join("static");
+    return "static";
   } else {
-    return path.join(__dirname, "..", "..", "..", "..", "static");
+    return path.join("..", "..", "..", "..", "static");
   }
 }
 
 function getFilesystemPathForStaticAsset(asset) {
   if (isDevelopment) {
-    return path.join(
-      ipcRenderer.sendSync("file.get-application-root"),
```

Comment:
I have some recollection that __dirname doesn't work in the renderer in production mode any more. is there any hint from the commit message about why I did that?

But if we don't need the rpc, great -- and we should remove the api on the server-side process as well